soh/soh/Enhancements/randomizer/randomizer_item_tracker.cpp

```diff
@@ -412,11 +412,14 @@ ItemTrackerNumbers GetItemCurrentAndMax(ItemTrackerItem item) {
 
 void DrawItemCount(ItemTrackerItem item) {
     int iconSize = CVarGetInteger("gItemTrackerIconSize", 36);
+    int textSize = CVarGetInteger("gTrackers.ItemTracker.ItemTrackerTextSize", 50);
     ItemTrackerNumbers currentAndMax = GetItemCurrentAndMax(item);
     ImVec2 p = ImGui::GetCursorScreenPos();
     int32_t trackerNumberDisplayMode = CVarGetInteger("gItemTrackerCapacityTrack", ITEM_TRACKER_NUMBER_CURRENT_CAPACITY_ONLY);
     int32_t trackerKeyNumberDisplayMode = CVarGetInteger("gItemTrackerKeyTrack", KEYS_COLLECTED_MAX);
 
+    ImGui::SetWindowFontScale(textSize / 50.0f);
```

Comment:
Where did the `50.0f` come from? 50 feels arbitrary to me with no relation to font sizes.

The base/default font size is `13.0f` for 13pt font size. I think this slider should be a float slider and depict the font size in point values, defaulting to 13. Then here you can divide by `13.0f` to get the scale value.

Reply:
> Where did the `50.0f` come from? 50 feels arbitrary to me with no relation to font sizes.
> 
> The base/default font size is `13.0f` for 13pt font size. I think this slider should be a float slider and depict the font size in point values, defaulting to 13. Then here you can divide by `13.0f` to get the scale value.

It didn't seem to matter what value was used, so I just picked one. I have no idea what font default size is haha. But yeah, I can change it